pkg/execution/state/redis_state/lua/includes/enqueue_to_partition.lua

```diff
@@ -8,7 +8,7 @@ local function enqueue_get_partition_item(partitionKey, id)
 end
 
 local function enqueue_to_partition(keyPartitionSet, partitionID, partitionItem, keyPartitionMap, keyGlobalPointer, keyGlobalAccountPointer, keyAccountPartitions, queueScore, queueID, partitionTime, nowMS)
-	if partitionID == "" then
+	if partitionID == "" or partitionID == nil or partitionID == false then
```

Comment:
partitionID will always be set to an (empty) string, so the nil or false case will not happen in the current implementation, but it's good to have a safeguard here.